app/init.php

```diff
@@ -81,7 +81,7 @@
 const APP_USERAGENT = APP_NAME . '-Server v%s. Please report abuse at %s';
 const APP_MODE_DEFAULT = 'default';
 const APP_MODE_ADMIN = 'admin';
-const APP_PAGING_LIMIT = 12;
+const APP_PAGING_LIMIT = 1;
```

Comment:
Is this change intentional?

Reply:
Gonna keep it for testing purposes, will remove at the end. Keeping Christy's comment opened so I don't forget.